### PR TITLE
Things

### PR DESCRIPTION
**Pull request by benhylau**
This addresses all the little things that I always wanted to add. I don't have hardware with me to test at the moment, so if someone wants to try it out, this is how the new branch-specific testing is supposed to work in my mind:

```
$ wget https://raw.githubusercontent.com/tomeshnet/prototype-cjdns-pi2/things/scripts/install && chmod +x install && TAG_PROTOTYPE_CJDNS_PI2=things WITH_MESH_POINT=true WITH_WIFI_AP=true WITH_IPFS=true ./install
```

cc. @stigtsp

### CODE REVIEW

File: scripts/install [hunk 1]
```diff
@@ -2,123 +2,28 @@
...
-fi
+# Export environment variables
+export WITH_MESH_POINT
+export WITH_WIFI_AP
+export WITH_IPFS
 
-# Reboot device
-sudo reboot
+# Run the actual installation script
+./install2
```
Comment (garrying) [inline scripts/install]: `./install2` is in `scripts`. https://github.com/tomeshnet/prototype-cjdns-pi2/blob/things/scripts/install#L20 should probably `cd` into it?

File: scripts/install [hunk 2]
```diff
@@ -17,7 +17,7 @@ fi
 if [ -z "$TAG_PROTOTYPE_CJDNS_PI2" ]; then
     TAG_PROTOTYPE_CJDNS_PI2=master
 fi
-cd prototype-cjdns-pi2
+cd prototype-cjdns-pi2/scripts
```
Comment (garrying) [inline scripts/install]: This breaks some of the install paths in the `install2`.

**Comment by benhylau** [comment]
@garrying I feel like cd-ing into `scripts` and removing that from the paths in `install2` is cleaner?

**Comment by garrying** [comment]
@benhylau Sure. I'm cool with that

**Pull request merged**